service/routes.py

```
@@ -4,7 +4,7 @@
 This microservice handles the lifecycle of Accounts
 """
 # pylint: disable=unused-import
-from flask import jsonify, request, make_response, abort, url_for   # noqa; F401
+from flask import jsonify, request, make_response, abort, url_for  # noqa: F401
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)